### PR TITLE
Add an example using pipe() to the file server demo

### PR DESCRIPTION
**Pull request by pmarkee**
IoT.js-Samples-DCO-1.0-Signed-off-by: Peter Marki marpeter@inf.u-szeged.hu

### CODE REVIEW

**Comment by pmarkee** [comment]
The WIP will be removed once the related patches land in iotjs.
https://github.com/Samsung/iotjs/pull/1784
https://github.com/Samsung/iotjs/pull/1800

**Comment by pmarkee** [comment]
I just realized that this PR has been open for months. Is it still needed?

File: file-server/client_with_pipe.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      response.unpipe(outputStream);
+    });
+  }).on('error', function() {
+    console.log(arguments);
+    console.log(arguments[0]);
+  });
+}
+
+//download('https://upload.wikimedia.org/wikipedia/commons/6/6a/Tizen_Logo.png', 'logo.png')
+download('http://localhost:8822/logo.png', 'foo.png');
```
Comment (LaszloLango) [inline file-server/client_with_pipe.js]: Missing new line

**Comment by pmarkee** [comment]
Thanks @LaszloLango , updated.

**Pull request merged**